modules/application/application.js

```diff
@@ -0,0 +1,35 @@
+class Application {
+  constructor() {
```

Comment:
- [ ] please export this class in order to make your application work as the previous CR already mentioned, what you need to do is add the words `export default` on line one so it will look like below:
```
export default class Application {
  "your code"
}
```

if you need more help with this please reach out to me via Slack, my full name is Michael Threels

Reply:
Done.